src/bramble-live-dev-cache-sw.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * See src/filesystem/impls/filer/UrlCache.js. If the browser supports
+ * CacheStorage and Service Worker, we cache request/response pairs
+ * for all files in the filesystem.  Each project root gets its own cache,
+ * which is named vfs/project/root.
+ */
+
+self.addEventListener("fetch", function(event) {
+    "use strict";
+
+    // Strip params off URL so it will properly match what's in cache
+    var url = event.request.url.split(/[?#]/)[0];
```

Comment:
since `URL` has been around since the Service Worker API has been around, you can use `URL` without worrying about poor browser support:

```js
var url = new URL(window.location.href).pathname;
```